packages/create-platformatic/src/create-package-json.mjs

```diff
@@ -20,6 +21,13 @@ const packageJsonTemplate = async (addTSBuild, fastifyVersion, platVersion) => {
     }
   }
 
+  if (projectType === 'db') {
+    pkg.dependencies = {
+      ...pkg.dependencies,
+      '@platformatic/db': `^${platVersion}`
```

Comment:
I would prefer if this `if` is moved in `create-db-cli` instead.

Reply:
This would result in this, to check that the property exists.

```javascript
    Object.assign(pkg.scripts, scripts)

    if (pkgOptions) {
      if ('dependencies' in pkgOptions) {
        Object.assign(pkg.dependencies, pkgOptions.dependencies)
      }
      if ('devDependencies' in pkgOptions) {
        Object.assign(pkg.devDependencies, pkgOptions.devDependencies)
      }
    }
```

The `createPackageJson` function is now having to do checks. Perhaps the best way to do this is to pass the `dependencies` option on its own. Like it is done with scripts option.

```javascript
const scripts = {
    migrate: 'platformatic db migrations apply'
}

const dependencies = {
  '@platformatic/db': `^${version}`
}
```

and then in the createPackageJson  function 

```javascript
...

  if (!isPackageJsonExists) {
    const pkg = await packageJsonTemplate(addTSBuild, fastifyVersion, platVersion)
    Object.assign(pkg.scripts, scripts)
    Object.assign(pkg.dependencies, dependencies)
    // if (pkgOptions) {
    //   if ('dependencies' in pkgOptions) {
    //     Object.assign(pkg.dependencies, pkgOptions.dependencies)
    //   }
    //   if ('devDependencies' in pkgOptions) {
    //     Object.assign(pkg.devDependencies, pkgOptions.devDependencies)
    //   }
    // }
    await writeFile(packageJsonFileName, JSON.stringify(pkg, null, 2))
    logger.debug(`${packageJsonFileName} successfully created.`)
  }

...
```